managed-identity.tf

```diff
@@ -1,5 +1,8 @@
 locals {
   managed_identity_list = toset(compact(concat(var.managed_identity_object_ids, [var.managed_identity_object_id])))
+  namespace             = var.namespace != null ? var.namespace : var.product
+  environment           = var.businessArea == "SDS" ? (var.env == "perftest") ? "test" : (var.env == "aat") ? "stg" : (var.env == "preview") ? "dev" : "${var.env}" : "${var.env}"
```

Comment:
this doesn't look right

`if businessArea = SDS, then if equal perftest override to test.`

it will never equal SDS and perftest

Below syntax is deprecated:
> `"${var.env}" : "${var.env}"`

It should be:
var.env.

This code in general is quite difficult to read and should be simplified

Reply:
Will alter the `"${var.env}" : "${var.env}"` 

On the first point - it's checking `var.env` instead of businessArea in that second check, as it has different env names in the cluster names - so if `businessArea == "SDS"` then it overrides to test etc - will try to look at simplifying somehow though